File: Discord_android/Discord/app/src/main/java/com/peanut/discord/models/ServerUser.java
package com.peanut.discord.models;
public class ServerUser {
    private int serverId;
    private int userId;
    private int roleId;
    private Server server;
    private User user;
    private Role role;
    public int getServerId() {
        return serverId;
    }
    public void setServerId(int serverId) {
        this.serverId = serverId;
    }
    public int getUserId() {
        return userId;
    }
    public void setUserId(int userId) {
        this.userId = userId;
    }
    public int getRoleId() {
        return roleId;
    }
    public void setRoleId(int roleId) {
        this.roleId = roleId;
    }
    public Server getServer() {
        return server;
    }
    public void setServer(Server server) {
        this.server = server;
    }
    public User getUser() {
        return user;
    }
    public void setUser(User user) {
        this.user = user;
    }
    public Role getRole() {
        return role;
    }
    public void setRole(Role role) {
        this.role = role;
    }
}

File: Server/Server/Models/ServerUser.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace Server.Models {
    public class ServerUser {
        public int ServerId { get; set; }
        public int UserId { get; set; }
        [Required]
        public int RoleId { get; set; }


        public Server Server { get; set; }
        public User User { get; set; }
        public Role Role { get; set; }
    }
}